### PR TITLE
Update cryptr.bash

### PR DESCRIPTION
**Pull request by Gu1llaum-3**
Ask the user if they want to delete the original file

```bash
cryptr encrypt test.txt
enter AES-256-CBC encryption password:
Verifying - enter AES-256-CBC encryption password:
Do you want to delete the original file? (y/N): y
[notice] Deleting the original file
```

### CODE REVIEW

**Comment by nodesocket** [comment]
Thanks for this PR. See my comments inline.

File: cryptr.bash [hunk 1]
```diff
@@ -50,6 +50,19 @@ cryptr_encrypt() {
   else
     openssl $OPENSSL_CIPHER_TYPE -salt -pbkdf2 -in "$_file" -out "${_file}.aes"
   fi
+
+  if [[ $? -eq 0 ]]; then
+    read -p "Do you want to delete the original file? (y/N): " confirm
+    if [[ "$confirm" =~ ^[Yy]$ ]]; then
+      echo "[notice] Deleting the original file"
```
Comment (nodesocket) [inline cryptr.bash:57]: Lowercase `D` to `d`.

File: cryptr.bash [hunk 2]
```diff
@@ -50,6 +50,19 @@ cryptr_encrypt() {
...
+      rm -f "$_file"
+    else
```
Comment (nodesocket) [inline cryptr.bash:59]: Remove the else and echo notice about original file not deleted.

File: cryptr.bash [hunk 3]
```diff
@@ -50,6 +50,19 @@ cryptr_encrypt() {
...
+      echo "[notice] Original file not deleted"
+    fi
+  else
+    echo "[error] Encryption failed, original file not deleted" 1>&2
```
Comment (nodesocket) [inline cryptr.bash:63]: Lowercase `E` to `e`.

**Review by Gu1llaum-3 — COMMENTED** [review]
Hi ! I have made the requested changes.

**Comment by nodesocket** [comment]
@Gu1llaum-3 just release `2.3.0` with your change. Thanks.

https://github.com/nodesocket/cryptr/commit/e1c18c2b4438032a2e8ba8960b738f6377f33d5c

**Pull request merged**